File: Labs/Exercise-06-after/sln/CookBook.Models/Recipe/RecipeListIngredientModel.cs
using EnumsNET;
using System;

namespace CookBook.Models
{
    public class RecipeListIngredientModel
    {
        public IngredientListModel Ingredient { get; set; }
        public double Amount { get; set; }
        public Unit Unit { get; set; }
        public string UnitText => Unit.AsString(EnumFormat.Description);
    }
}

File: Lectures/Lecture_11/CookBook/src/app/api/models/recipe-list-ingredient-model.ts
/* tslint:disable */
/* eslint-disable */
import { IngredientListModel } from './ingredient-list-model';
import { Unit } from './unit';
export interface RecipeListIngredientModel {
  amount?: number;
  ingredient?: null | IngredientListModel;
  unit?: Unit;
  unitText?: null | string;
}
